service/prometheus/service.go

```diff
@@ -0,0 +1,82 @@
+package prometheus
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"path"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+)
+
+const (
+	prometheusReloadPath = "/-/reload"
```

Comment:
I would appreciate a descriptive godoc here because I have no idea what this is about and what e.g. the dash means.

Reply:
Done.